src/main/java/io/confluent/clientmetrics/MetricsPublisher.java

```diff
@@ -0,0 +1,102 @@
+package io.confluent.clientmetrics;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.confluent.clientmetrics.avro.AVROMetricRecords;
+import io.confluent.clientmetrics.avro.AVROMetricValue;
+import org.apache.kafka.clients.producer.KafkaProducer;
+import org.apache.kafka.clients.producer.ProducerRecord;
+import org.apache.kafka.common.Metric;
+import org.apache.kafka.common.MetricName;
+
+import java.util.Map;
+import java.util.function.Supplier;
+import java.util.stream.Collectors;
+
+/**
+ * Metrics publisher can be scheduled to run periodically, like:
+ * <p>
+ * final ScheduledExecutorService executorService = Executors.newScheduledThreadPool(1);
+ * executorService.scheduleAtFixedRate(() -> sendMetrics(producer, streamsClient), 0, 1, TimeUnit.SECONDS);
```

Comment:
I dont understand this javadoc?

Reply:
Just an example of how to run it scheduled. Do you want me to remove it?

Reply:
But the MetricsPublisher isn't mentioned? just the sendMetrics method? which implies it's how it's to be used from within the class. Which given that it's in the class javadoc, breaks the encapsulation of the class.. Is it in the wrong place? Or is it missing something?

Reply:
Does this not run periodically by itself anymore?